draft/0000-translatable-strings-rule.md

```diff
@@ -31,6 +31,7 @@ Enless discussions create friction and I'm sure we want everything but that for
 ### 1. Do not modify translatable strings in PHP files, only in CSV files
 
 **PRO**: It doesn't break language packs, which, important to re-state, we do not have control over.
```

Comment:
another PRO, its in sync with translation standards which are usually *key* or *id* based